cmd/query/app/query_service.go

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2019 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package app
+
+import (
+	"github.com/opentracing/opentracing-go"
+	"go.uber.org/zap"
+
+	"github.com/jaegertracing/jaeger/model/adjuster"
+	"github.com/jaegertracing/jaeger/storage/dependencystore"
+	"github.com/jaegertracing/jaeger/storage/spanstore"
+)
+
+// QueryService contains span utils required by the query-service.
+type QueryService struct {
```

Comment:
This is a good start, but the idea is to encapsulate everything into this type, not keep accessing its private members from the handlers. To help with that I would recommend moving it to its own package `app/querysvc`.